File: Programmars/Level_1/가운데 글자 가져오기.py
def solution(s):
    half = len(s) // 2
    if len(s) % 2 != 0:
        answer = s[half]
    else:
        answer = s[half-1:half+1]
    return answer


print(solution("abcde"))  # "c"
print(solution("qwer"))  # "we"

File: JavaScript/Programmars/Level_1/가운데 글자 가져오기.js
// 내 코드
function solution(s) {
  let answer = '';
  const sLen = s.length;
  const midIdx = Math.floor(sLen / 2);

  if (sLen % 2 === 1) answer = s[midIdx];
  else answer = s.substr(midIdx - 1, 2);

  return answer;
}

// 다른 코드
function solution(s) {
  return s.substr(Math.ceil(s.length / 2) - 1, s.length % 2 === 0 ? 2 : 1);
}
